### ISSUE TITLE
You did not follow semantic versioning

### ISSUE THREAD

**Issue by halloei** [svenluijten/env-providers#8]
Your commit https://github.com/svenluijten/env-providers/commit/cf7ea2baea9a9b547be074aa894ff32f4e401563 broke backwards compatibility (you renamed `\Sven\EnvProviders\EnvServiceProvider` to `\Sven\EnvProviders\ServiceProvider`).  But this commit was included in a minor update of your package. Version 3.1.0 should have been version 4.0.0.
Please follow semantic versioning in the future to prevent our applications to break.

**Comment by halloei** [comment]
You forgot to update the package discovery config for version 3.1.1, which is currently the problem:
```
"extra": {
    "laravel": {
      "providers": [
        "Sven\\EnvProviders\\ServiceProvider"
      ]
    }
  }
```

**Comment by svenluijten** [comment]
Hi @halloei,

Yeah, this was a problem and I skipped over the change in `composer.json` in my rush to publish the fix (v3.1.1). I just changed the provider's name in `composer.json` and tagged [v3.1.2](https://github.com/svenluijten/env-providers/releases/tag/v3.1.2). Sorry about the trouble!
